notebooks/Notebook2.md

```diff
@@ -17,18 +12,34 @@ exciting milestones, preliminary results, etc. Feel free to include images
 (e.g., a sketch of the design or a screenshot of a running program), links to
 code, and any other resources that you think will help clearly convey your
 design process.
+-- REMOVE 
+Some reps to look at https://github.com/hmc-cs111-spring2023/hw-7-piconot-rydermitchell-hw7/blob/main/build.md
+https://github.com/hmc-cs111-spring2023/hw-7-sample-solution/tree/main/src/main/scala/piconot/external
+
+--
+
+This week I did not work as much as I would have liked. I mainly just took our sample solution from hw7 and converted it into my artifact repo. I want to use its external solution as structure for my own.  The main progress I made was on a language for parseing my language. Check it out in artifact [design.md](https://github.com/hmc-cs111-spring2023/artifact-AdamrBeckwith/blob/main/design.md) would love feedback on that setup. (I am considering the writing there as part of this notebook) 
 
 ## Questions
 
 **What is the most pressing issue for your project? What design decision do
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
 
+I need to work more. That is the biggest issue. Im struggling to *do* so maybe I need to define my direction better. 
+
+
 **What questions do you have for your critique partners? How can they best help
 you?**
 
+I would love any feedback on the structural design I have so far, and any tips for going forward with a external language. 
```

Comment:
The way you've organized your project is phenomenal- I should definitely spend some time getting sbt and the packages set up in my project the way you have. We talked a bit about using regexes in class, which might be a possibility, but I think parser combinators will be a better way to go in general.

Reply:
It might be worth peeking at some old solution code for PLs- while it's all in Haskell it might have some good tips for how to lay out your AST, parser, and pretty printer. Speaking of pretty printers- have you given any thought to how the output of your language might look?